src/edge-core-index.js

```diff
@@ -323,6 +323,7 @@ export type EdgeAccountCallbacks = {
     walletId: string,
     abcTransactions: Array<EdgeTransaction>
   ) => mixed,
+  +onTxidsChanged?: (walletId: string, txids: Array<string>) => mixed,
```

Comment:
I don't believe the GUI should need this callback since it already gets the onTransactionsChanged.